frontend/integrations/meetup/utilities.js

```diff
@@ -0,0 +1,74 @@
+import { circleOfCoords } from '../../utilities/coords';
+
+export const groupParser = (m) => {
+  const id = m.id;
+  m = m.data;
+  return {
+    id,
+    name: m.name,
+    latlng: {
+      lat: m.lat,
+      lng: m.lon
+    },
+    group_photo: m.group_photo ? m.group_photo.photo_link : undefined,
+    key_photo: m.key_photo ? m.key_photo.photo_link : undefined,
+    members: m.members,
+    location: m.localized_location,
+    link: m.link,
+    urlname: m.urlname,
+    description: m.description
+  };
+};
+
+export const eventParser = e => {
+  const id = e.id;
+  const date = e.date;
+  e = e.data;
+  return {
+    id,
+    date,
+    link: e.link,
+    name: e.name,
+    latlng: e.venue ? { lat: e.venue.lat, lng: e.venue.lon } : undefined,
+    venue: e.venue ? {
+      address_1: e.venue.address_1,
+      zip: e.venue.zip,
+      city: e.venue.city
+    } : undefined,
+    time: e.time,
+    duration: e.duration,
```

Comment:
if this is not specified it should default to 3 hours, as per meetup [api docs](https://www.meetup.com/meetup_api/docs/:urlname/events/#list):
"Scheduled event duration in milliseconds, if an end time is specified by the organizer. When not present, a default of 3 hours may be assumed by applications"

Reply:
Good point, adding this